bin/setup

```diff
@@ -4,6 +4,7 @@
 # https://github.com/thoughtbot/guides/tree/master/protocol
 
 require 'pathname'
+require 'yaml'
```

Comment:
Also in #435 . No preference which goes in 🤷  (cc @orenyk)

Reply:
oh I didn't check the diff last night, I just didn't see a separate PR for it so I opened my own. I'm happy to just close mine to keep things simple 😄